examples/js/loaders/GLTFLoader.js

```diff
@@ -1088,14 +1088,26 @@ THREE.GLTFLoader = ( function () {
 
 					materialParams.emissive = new THREE.Color().fromArray( materialValues.emission );
```

Comment:
Hm. Actually I think this code should be using `emissive` / `emissiveMap` for Phong materials, and `color` / `map` for MeshBasicMaterial. Let me rework this a little.